projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -40,34 +44,53 @@ const ADD_MEDIA_LABEL = __( 'Set Social Image', 'jetpack' );
 const REPLACE_MEDIA_LABEL = __( 'Replace Social Image', 'jetpack' );
 const REMOVE_MEDIA_LABEL = __( 'Remove Social Image', 'jetpack' );
 
-const ALLOWED_MEDIA_TYPES = [ 'image/jpeg', 'image/png' ];
-
 /**
  * Wrapper that handles media-related functionality.
  *
  * @returns {object} The media section.
  */
 export default function MediaSection() {
+	const [ showNotice, setShotNotice ] = useState( false );
 	const { attachedMedia, updateAttachedMedia } = useAttachedMedia();
+	const { connections } = useSocialMediaConnections();
```

Comment:
I think we have the enabled connections coming back from the hook.

```suggestion
	const { enabledConnections } = useSocialMediaConnections();
```

Reply:
Yeah, we have but that one maps it after filtering, and only gives back the ID-s. The same goes for `skippedConnections`. We could change that, but I saw that it is used in some places.

Reply:
I changed the declaration of it, now we can use it from the hook